index.yaml

```diff
@@ -21,6 +21,13 @@ apps:
     description: "Cilium is an open-source, cloud-native tool for managing, securing, and monitoring network connectivity between workloads, powered by the innovative Kernel technology eBPF."
     category: Observability
 
+  - name: cluster-autoscaler-aws
+    displayName: AWS Cluster Autoscaler
+    website: "https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler"
+    imageUrl: "https://raw.githubusercontent.com/kubefirst/gitops-catalog/main/logos/cluster-autoscaler.svg"
+    description: "Configured for clusters running on AWS. Cluster Autoscaler is a tool that automatically adjusts the size of the Kubernetes cluster when one of the following conditions is true: 1) there are pods that failed to run in the cluster due to insufficient resources, or 2) there are nodes in the cluster that have been underutilized for an extended period of time and their pods can be placed on other existing nodes."
```

Comment:
Description need to be 200 characters or less (UI restriction). https://github.com/konstructio/gitops-catalog/blob/main/CONTRIBUTING.md#add-your-application